Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/RobotCameraPipeline.java

```diff
@@ -0,0 +1,80 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.imgproc.Moments;
+import org.openftc.easyopencv.OpenCvPipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RobotCameraPipeline extends OpenCvPipeline {
+    public double xPosition = 0.0;
+    public double yPosition = 0.0;
+    public double width = 0.0;
+
+    private Scalar lowerRange;
+    private Scalar upperRange;
+
+    List<MatOfPoint> contours = new ArrayList<>();
+    Mat hierarchy = new Mat();
+    Mat mat = new Mat();
+
+    // method used to set the ranges for the pipeline
```

Comment:
Technically this should be a javadoc comment not a single line comment.

Reply:
@SJJCoding